ansul/Assignment 3/MainActivity.java

```diff
@@ -0,0 +1,65 @@
+package com.android.fcc.activitylifecycle;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        displayToast("Activity 1 onCreate");
+        setContentView(R.layout.activity_main);
+        Button button=(Button) findViewById(R.id.activity2_button);
+        button.setOnClickListener(new View.OnClickListener(){
+            @Override
+            public void onClick(View v){
+                try{
+                    Intent intent = new Intent(MainActivity.this, SecondActivity.class);
+                    displayToast("Going to activity 2");
+                    startActivity(intent);
+                }catch (Exception e){
```

Comment:
Its generally a bad idea to include whole Exception class like this also if you are planning to handle them , then be sure to do that.